src/Analyzers/CSharp/Tests/RemoveUnusedParametersAndValues/RemoveUnusedValueAssignmentTests.cs

```diff
@@ -2420,7 +2420,7 @@ void M(object p)
                 break;
         };
     }
-}", options: PreferDiscard);
+}", options: PreferDiscard, parseOptions: new CSharpParseOptions(LanguageVersion.CSharp8));
```

Comment:
What is the behavior for the test with `CSharp9`?

Reply:
Added a test with `CSharp9`. The refactoring/fixer uses a type pattern in that case (ie. `case int:` instead of `case int _:`).

---
In reply to: [470298910](https://github.com/dotnet/roslyn/pull/46645#discussion_r470298910) [](ancestors = 470298910)